.github/workflows/deploy.yml

```
@@ -155,6 +171,66 @@ jobs:
+
+  # ──────────────────────────────────────────────────────────────────────────
+  # Job: Ent schema migrate (must pass before ECS deploy jobs)
+  # ──────────────────────────────────────────────────────────────────────────
+  migrate-ent:
+    name: Ent DB migrations
+    needs: [resolve-config]
+    if: ${{ github.event.inputs.dry_run != 'true' && needs.resolve-config.result == 'success' }}
+    runs-on: self-hosted
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)